janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/compat/AbstractESCompat.java

```diff
@@ -72,9 +74,16 @@ public String scriptLang() {
         return "painless";
     }
 
+    public ImmutableMap.Builder prepareScript(String inline, List<Map<String, Object>> fields) {
```

Comment:
I think we can rename `inline` argument to `source`

Reply:
done